modules/pipeline/pipengine/engine.go

```diff
@@ -96,42 +99,6 @@ func (engine *Engine) WaitDBGC(pipelineID uint64, ttl uint64, needArchive bool)
 	engine.reconciler.WaitDBGC(pipelineID, ttl, needArchive)
 }
 
-const logPrefixContinueLoading = "continue load running pipelines"
-
-func (engine *Engine) continueLoadRunningPipelines() {
```

Comment:
Moved to where?

Reply:
![image](https://user-images.githubusercontent.com/30427474/132154161-427b0388-5947-4126-aa36-905115d99d72.png)
No usages found in All Places, Its function is to import the running pipeline,engine.loadRunningPipelines() already carried out

Reply:
i restored it, may use it later